test/functional/test_framework/address.py

```diff
@@ -159,6 +162,16 @@ def check_script(script):
     assert False
 
 
+def bech32_to_bytes(address):
+    hrp = address.split('1')[0]
+    if hrp not in ['bc', 'tb', 'bcrt']:
```

Comment:
I think this is totally fine as is here for now, but in the future we may want to put these all in a list/enum